diff --git a/node_modules/react-native-safe-area-context/android/build.gradle b/node_modules/react-native-safe-area-context/android/build.gradle
index f324b9a..567be3f 100644
--- a/node_modules/react-native-safe-area-context/android/build.gradle
+++ b/node_modules/react-native-safe-area-context/android/build.gradle
@@ -41,9 +41,9 @@ if (isNewArchitectureEnabled()) {
 android {
     def agpVersion = com.android.Version.ANDROID_GRADLE_PLUGIN_VERSION
     if (agpVersion.tokenize('.')[0].toInteger() >= 7) {
-        namespace "com.th3rdwave.safeareacontext"
+        namespace = "com.th3rdwave.safeareacontext"
         buildFeatures {
-            buildConfig true
+            buildConfig = true
         }
     }

@@ -55,7 +55,7 @@ android {
         ndkPath rootProject.ext.ndkPath
     }
     if (rootProject.hasProperty("ndkVersion")) {
-        ndkVersion rootProject.ext.ndkVersion
+        ndkVersion = rootProject.ext.ndkVersion
     }

     defaultConfig {
@@ -71,7 +71,7 @@ android {
     }

     lintOptions{
-        abortOnError false
+        abortOnError = false
     }

     packagingOptions {
@@ -111,7 +111,7 @@ repositories {
     google()
     maven {
         // All of React Native (JS, Obj-C sources, Android binaries) is installed from npm
-        url "$rootDir/../node_modules/react-native/android"
+        url = "$rootDir/../node_modules/react-native/android"
     }
     mavenCentral()
 }
diff --git a/node_modules/react-native-safe-area-context/android/src/main/AndroidManifest.xml b/node_modules/react-native-safe-area-context/android/src/main/AndroidManifest.xml
index f5da598..ae0b1d0 100644
--- a/node_modules/react-native-safe-area-context/android/src/main/AndroidManifest.xml
+++ b/node_modules/react-native-safe-area-context/android/src/main/AndroidManifest.xml
@@ -1,5 +1,2 @@

-<manifest
-	xmlns:android="http://schemas.android.com/apk/res/android"
-  package="com.th3rdwave.safeareacontext">
-</manifest>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"></manifest>
diff --git a/node_modules/react-native-safe-area-context/android/src/main/java/com/th3rdwave/safeareacontext/SafeAreaView.kt b/node_modules/react-native-safe-area-context/android/src/main/java/com/th3rdwave/safeareacontext/SafeAreaView.kt
index b29e787..d8e4322 100644
--- a/node_modules/react-native-safe-area-context/android/src/main/java/com/th3rdwave/safeareacontext/SafeAreaView.kt
+++ b/node_modules/react-native-safe-area-context/android/src/main/java/com/th3rdwave/safeareacontext/SafeAreaView.kt
@@ -1,3 +1,5 @@
+@file:Suppress("DEPRECATION")
+
 package com.th3rdwave.safeareacontext

 import android.content.Context
diff --git a/node_modules/react-native-safe-area-context/android/src/main/java/com/th3rdwave/safeareacontext/SafeAreaViewShadowNode.kt b/node_modules/react-native-safe-area-context/android/src/main/java/com/th3rdwave/safeareacontext/SafeAreaViewShadowNode.kt
index 939950a..a8de1dd 100644
--- a/node_modules/react-native-safe-area-context/android/src/main/java/com/th3rdwave/safeareacontext/SafeAreaViewShadowNode.kt
+++ b/node_modules/react-native-safe-area-context/android/src/main/java/com/th3rdwave/safeareacontext/SafeAreaViewShadowNode.kt
@@ -1,3 +1,5 @@
+@file:Suppress("DEPRECATION")
+
 package com.th3rdwave.safeareacontext

 import com.facebook.react.bridge.Dynamic
